.github/workflows/test.yml

```diff
@@ -0,0 +1,453 @@
+# This file was auto-generated by .evergreen/make-github-actions
+# at 2021-08-21T15:14:55Z
+---
+name: Run Mongoid Tests
+'on':
+- push
+- pull_request
+jobs:
+  build:
+    name: "${{matrix.ruby}} driver-${{matrix.driver}} mongodb-${{matrix.mongodb}}
+      ${{matrix.topology}}"
+    env:
+      CI: true
+      TESTOPTS: "-v"
+    runs-on: ubuntu-latest
+    continue-on-error: "${{matrix.experimental}}"
```

Comment:
Oh, I totally forgot about this one - I was wondering how to solve experimental builds (e.g. testing against new language versions). This may ensure that we're always testing builds we expect to be stable and could add them as required checks for merging.